ibtk/include/ibtk/HierarchyIntegrator.h

```diff
@@ -885,28 +886,30 @@ class HierarchyIntegrator : public SAMRAI::mesh::StandardTagAndInitStrategy<NDIM
     /*
      * Time and time step size data read from input or set at initialization.
      */
-    double d_integrator_time, d_start_time, d_end_time;
-    double d_dt_init, d_dt_min, d_dt_max, d_dt_growth_factor;
-    int d_integrator_step, d_max_integrator_steps;
+    double d_integrator_time = std::numeric_limits<double>::quiet_NaN(), d_start_time = 0.0,
+           d_end_time = std::numeric_limits<double>::max();
+    double d_dt_init = std::numeric_limits<double>::max(), d_dt_min = 0.0,
+           d_dt_max = std::numeric_limits<double>::max(), d_dt_growth_factor = 2.0;
+    int d_integrator_step = 0, d_max_integrator_steps = std::numeric_limits<double>::max();
```

Comment:
This cannot be the correct value for `d_max_integrator_steps` since it causes an overflow (one is `int` and the other is `double`).

Reply:
Yes. It should be `std::numeric_limits<int>::max()`. I'll fix this once you find all of my mistakes...